apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -14,81 +14,88 @@ const styles = {
     marginBottom: 20,
     display: 'flex',
     flexFlow: 'wrap',
-    boxSizing: 'border-box',
-    color: color.charcoal
+    boxSizing: 'border-box'
   },
   lessonDetails: {
-    width: '50%',
+    width: '75%',
     margin: 15
   },
-  button: {
-    marginLeft: 25,
-    marginRight: 25,
-    marginTop: 18,
-    marginBottom: 18
-  },
   lessonLevel: {
     fontSize: 16,
-    marginBottom: 10
+    marginBottom: 8,
+    color: color.teal
+  },
+  unit: {
+    color: color.teal
   },
   label: {
     fontFamily: '"Gotham 5r", sans-serif',
     marginRight: 5,
     marginLeft: 10
   },
   time: {
-    width: '25%',
-    marginTop: 30,
-    fontStyle: 'italic'
+    marginTop: 15,
+    fontStyle: 'italic',
+    color: color.light_gray,
+    float: 'left'
+  },
+  comment: {
+    width: '100%',
+    fontStyle: 'italic',
+    color: color.charcoal,
+    marginLeft: 25,
```

Comment:
nit: simplify lines 46-48 to `margin: '0 25 15 25'` (0 may or may not work for marginTop)